src/DB/DeleteQueryWriter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DB;
+
+class DeleteQueryWriter implements QueryWriterInterface
+{
+    public function writeQuery(QueryInfo $info): string
+    {
+        $table = $info->getTable();
+
+        if ($table === null) {
+            throw new \InvalidArgumentException('Invalid delete query structure');
+        }
+
+        $delete = 'DELETE FROM ' . $table;
```

Comment:
__optional__

For me, personally it is easier to read when using string interoplation, but you could do as you wish. 😄